src/config.ts

```diff
@@ -190,7 +192,12 @@ export class KubeConfig {
         this.users = [
             {
                 name: userName,
-                token: fs.readFileSync(`${pathPrefix}${Config.SERVICEACCOUNT_TOKEN_PATH}`).toString(),
```

Comment:
@brendanburns What happens in the case that still is still a valid form of authentication?

Does it feel like we are removing this functionality of being able to raw pass just a token we should be updating more tests/code to clean up the usage of `user.token`